src/commands/unity-symbols/upload.ts

```diff
@@ -0,0 +1,255 @@
+import fs from 'fs'
+import path from 'path'
+
+import {Command, Option} from 'clipanion'
+
+import {newApiKeyValidator} from '../../helpers/apikey'
+import {RepositoryData, getRepositoryData, newSimpleGit} from '../../helpers/git/format-git-sourcemaps-data'
+import {MetricsLogger, getMetricsLogger} from '../../helpers/metrics'
+import {MultipartValue, UploadStatus} from '../../helpers/upload'
+import {DEFAULT_CONFIG_PATHS, performSubCommand, resolveConfigFromFileAndEnvironment} from '../../helpers/utils'
+import * as validation from '../../helpers/validation'
+import {checkAPIKeyOverride} from '../../helpers/validation'
+import {version} from '../../helpers/version'
+
+import * as dsyms from '../dsyms/upload'
+
+import {getUnityRequestBuilder, uploadMultipartHelper} from './helpers'
+import {IL2CPP_MAPPING_FILE_NAME, MappingMetadata, TYPE_IL2CPP_MAPPING, VALUE_NAME_IL2CPP_MAPPING} from './interfaces'
+import {
+  renderArgumentMissingError,
+  renderFailedUpload,
+  renderGitWarning,
+  renderMissingBuildId,
+  renderMissingIL2CPPMappingFile as renderMissingIl2CppMappingFile,
+  renderRetriedUpload,
+  renderUpload,
+} from './renderer'
+
+export class UploadCommand extends Command {
+  public static paths = [['unity-symbols', 'upload']]
+
+  public static usage = Command.Usage({
+    category: 'RUM',
+    description: 'Upload Unity symbol files to Datadog.',
+    details: `
+            This command will upload all iOS symbol files for Unity applications in order to symbolicate errors and
+            crash reports received by Datadog. This includes uploading dSYMs and IL2CPP mapping files.
+        `,
+    examples: [
+      [
+        'Upload all symbol files from the default location',
+        'datadog-ci unity-symbols upload --service-name com.datadog.example',
```

Comment:
```suggestion
        'datadog-ci unity-symbols upload --service-name com.companyname.example',
```

Reply:
Actually I have to fix these examples. Service isn't necessary for `unity-symbols upload`.